documentation/modules/exploit/multi/browser/chrome_object_create.md

```diff
@@ -1,7 +1,10 @@
 This modules exploits a type confusion in Google Chromes JIT compiler. The Object.create operation can be used to cause a type confusion between a PropertyArray and a NameDictionary.
 The type confusion can be used to construct a arbitrary read/write memory primitive, which is used to write shellcode into rwx region of a WebAssembly object.
 
-**This module does not contain an exploit to escape the sandbox, so you must launch Google Chrome with the --no-sandbox option**
+This module can target the renderer process (target 0), but Google Chrome must be launched with the --no-sandbox flag for the payload to execute successfully.
+
+Alternatively, this module can use CVE-2019-1458 to escape the renderer sandbox (target 1). This will only work on vulnerable versions of Windows (e.g Windows 7) and the exploit can only be triggered once.
+Additionally the exploit can cause the target machine to restart when the session is terminated. A BSOD is also likely to occur when the system is shut down or rebooted.
```

Comment:
These lines look great but my concern is that you have individual lines that are way too long, when they should be broken up into multiple lines (even if they are within the same sentence). Can you run `tools/dev/msftidy_docs.rb` on this file and try address some of the concerns it raises? It should note a few things such as long lines like these that can be tidied up to help make the documentation easier to read.